src/utils/ledgerHash.ts

```diff
@@ -1,121 +1,88 @@
-import _ from 'lodash'
```

Comment:
Should this file be in the `hashes` folder?

Reply:
Yeah, this was a duplicate, deleting.